src/NuGet.Core/NuGet.Versioning/NuGet.Versioning.csproj

```diff
@@ -7,6 +7,8 @@
     <PackProject>true</PackProject>
     <Shipping>true</Shipping>
     <IncludeInVSIX>true</IncludeInVSIX>
+    <GenerateDocumentationFile Condition="'$(IsOfficialBuild)' == 'true'">false</GenerateDocumentationFile>
```

Comment:
@nkolev92 So from the previous PR, are you suggesting we enable document generation for all of them?

Reply:
Official builds create "real signed" binaries after a PR is merged. By disabling GenerateDocumentationFile for non-official builds, it means that pull requests can have green builds that break official builds when merged.  This absolutely should not be allowed.

Reply:
It should be the opposite; it's disabling document generation during official builds right now. Enabling them otherwise.

Reply:
Ok, you're right. But doesn't setting `GenerateDocumentFile` to false for official builds mean our nupkgs that we publish to nuget.org won't have the xmldoc, so people using our NuGet.Versioning package won't get these xmldoc intellisense?

Reply:
Oof yes! I missed the "PackProject" element. Gonna just set it to always generate the document file.